.github/workflows/allthekeeps-testnet.yml

```diff
@@ -0,0 +1,103 @@
+name: AllTheKeeps / Testnet
+
+on:
+  push:
+    branches:
+     - allthekeeps-on-ropsten
```

Comment:
We should use master (main).

Reply:
pls see this comment: https://github.com/keep-network/allthekeeps/pull/1#issuecomment-900255001

Reply:
Oh, now I see that  I had prepared a comment regarding https://github.com/keep-network/allthekeeps/pull/1#issuecomment-900255001 but it got lost.

We should keep everything in master branch. If it requires changes to support testnet along with the mainnet we should introduce them to make the master code more universal. We should consider contributing the changes to the original repository, so we don't face conflicts in the future.

Reply:
Ok, I'd rather not block this PR. I agree, that making it more universal would be better/cleaner, but I figured that if all we need is to make it work on Ropsten for now then it would be good enough for our purposes. The last commit to the original ATK was made 6months ago so it's not something that changes often. I see what I can do to make it more universal in the follow up PRs